docs/setup.md

```diff
@@ -0,0 +1,151 @@
+# Setup
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed on your system:
+
+- Node.js (v18)
+- npm (v8.3.0 or higher)
+- [Docker](https://www.docker.com/products/docker-desktop/) (v4.32.0 or higher)
+
+## Developer Setup
+
+This guide will walk you through setting up a complete development environment, including Arranger and its complementary services.
+
+### Setting up supporting services
+
+We'll use our Conductor service, a flexible Docker Compose setup, to spin up Maestro's complementary services.
```

Comment:
"we" here implies us + the users reading the guide, therefor "our" would mean Conductor is also theirs. Mine!
```suggestion
We'll use the Overture Conductor service, a flexible Docker Compose setup, to spin up Maestro's complementary services.
```